### PR TITLE
Release Conversations Language Understanding SDK 1.1.0-beta.1

### LINKED ISSUE #26379: Re-enable AnalyzeConversationWithLanguage tests when language vs. locale issue is resolves
The functional and sample tests named `AnalyzeConversationWithLanguage` and `AnalyzeConversationWithLanguageAsync` stopped working live with a rather vague 400 BadRequest. After some experimentation I found that BCP-47 language identifiers like "es" no longer work and a full locale identifier like "es-ES" is required. This is an unexpected changes and I want to clarify with the service team this is expected before making changes to the tests. We should also update the swagger documentation if that is the case.

For now I've disabled the tests as opposed to "fixing" them with what could be an accidental regression.

/cc @AhmedLeithy @ZulaMostafa @annatisch @mshaban-msft @ChongTang

### PR DESCRIPTION
**Pull request by heaths**
* Convert to DPG with HLC models
* Ignore long sync LRO test

  Caused by #29140 (seemingly)

* Add swagger transforms

  Works around #29141 and #29143

* Convert to DPG

  Also fixes #26379

* Update public APIs and documentation
* Resolve PR feedback
* Resolve offline feedback
* Update generated code

### CODE REVIEW

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                    "resolution",
+                }
+            },
+            kind = "ConversationalSummarizationTask",
+            taskName = "1",
+        },
+    },
+};
+
+Operation<BinaryData> analyzeConversationOperation = client.SubmitJob(WaitUntil.Started, RequestContent.Create(data));
```
Comment (kristapratico) [inline sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md:72]: I thought it was called AnalyzeConversation?

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    }
+}
+```
+
+## Asynchronous
+
+Using the same `data` definition above, you can make an asynchronous request by calling `AnalyzeConversationAsync`:
+
+```C# Snippet:AnalyzeConversationAsync_ConversationSummarization
+Operation<BinaryData> analyzeConversationOperation = await client.SubmitJobAsync(WaitUntil.Started, RequestContent.Create(data));
```
Comment (kristapratico) [inline sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md:101]: same here (and across all samples it looks like)

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md (hunk 1 above)
Comment (heaths) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md:72]: It is. I forgot to run our snippet updater.

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md (hunk 2 above)
Comment (heaths) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md:101]: Same resolution.

**Review by kristapratico — APPROVED** [review]

**Review by jsquire — APPROVED** [review]

**Pull request merged**